content/en/legacy/helm-operator/get-started/using-helm.md

```diff
@@ -114,4 +114,4 @@ in the `HelmRelease` guide.
   documentation](../references/chart.md#configuration)
   and [operator reference](../references/operator.md).
 - Continue learning about `HelmRelease` resources [in the
-  guide](../helmrelease-guide/introduction.md).
\ No newline at end of file
+  guide](../helmrelease-guide/introduction.md).
```

Comment:
Hum... what is this change about?

Reply:
@dholbach It appears that GitHub automatically added a newline at the end of this Markdown document.